package.json

```diff
@@ -32,8 +33,14 @@
       "git add"
     ]
   },
+  "peerDependencies": {
+    "@dojo/framework": "file:./tarballs/dojo-framework-3.0.0-pre.tgz",
+    "@dojo/webpack-contrib": "file:./tarballs/dojo-webpack-contrib-3.0.0-alpha.1.tgz"
+  },
   "devDependencies": {
-    "@dojo/cli": "2.0.0",
+    "@dojo/cli": "file:./tarballs/dojo-cli-3.0.0-alpha.1.tgz",
+    "@dojo/shim": "^2.0.0",
```

Comment:
Still needed?

Reply:
Evidently, no. At one point yesterday, I was adding things until it worked and this was one of those things.